README.md

```diff
@@ -1 +1,62 @@
-# microverse-week-1
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+> One paragraph statement about the project.
```

Comment:
- [ ] Kindly add a project summary to briefly explain what was done in the project😊